client/signup/steps/domains/index.jsx

```diff
@@ -605,6 +620,14 @@ export class RenderDomainsStep extends Component {
 				} ) );
 			}
 
+			await this.props.shoppingCartManager.addProductsToCart( [ registration ] ).catch( () => {
+				this.handleReplaceProductsInCartError(
+					this.props.translate(
+						'Sorry, there was a problem adding that domain. Please try again later.'
+					)
+				);
+			} );
```

Comment:
Does catch work with await like this? (TIL?) I would have expected a try catch around this.

Reply:
Yeah, we use that to deal with other server-side errors on domains like `myfinal.org`. More context here: p1702417233912589/1702403962.265209-slack-CKZHG0QCR